### PR TITLE
wait for cloud to actually stop the test on first '^C'

### PR DESCRIPTION
**Pull request by MStoykov**
Also:
- move the signal handling as far up as possible
- stop the test in goroutine so if that takes a while a second ^C will
  again stop the test

### CODE REVIEW

File: cmd/cloud.go [hunk 1]
```diff
@@ -213,6 +213,29 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
...
+			go func() {
+				sig := <-sigC
+				logger.WithField("sig", sig).Print("Stopping cloud test run in response to signal...")
+				// Do this in a separate goroutine so that if it blocks the second signal can stop the execution
+				go func() {
+					stopErr := client.StopCloudTestRun(refID)
+					if stopErr != nil {
+						logger.WithError(stopErr).Error("Stop cloud test error")
+					} else {
+						logger.Info("Successfully send segnal to stop test to the cloud, now waiting for it to actually stop.")
```
Comment (na--) [inline cmd/cloud.go:229]:
```suggestion
						logger.Info("Successfully sent signal to stop the cloud test, now waiting for it to actually stop...")
```

**Review by na-- — COMMENTED** [review]
LGTM besides the typo, also please close https://github.com/loadimpact/k6/pull/1704

**Review by na-- — APPROVED** [review]
LGTM

**Review by imiric — APPROVED** [review]
I didn't test it, but LGTM.

**Pull request merged**